src/connection_manager.c

```diff
@@ -157,6 +160,7 @@ static qd_config_sasl_plugin_t *qd_find_sasl_plugin(qd_connection_manager_t *cm,
     return 0;
 }
 
+// XXX Should this move to server.c or its own server_config.c?
```

Comment:
IIRC the qd_server_config_t structure is public - connection_manager and server (perhaps others) have direct access to its fields.  I prefer breaking these out to their own .h and .c files.